x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/cti/indicator_details.tsx

```diff
@@ -0,0 +1,117 @@
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  INDICATOR_DATASET,
+  INDICATOR_MATCHED_TYPE,
+  INDICATOR_PROVIDER,
+  INDICATOR_REFERENCE,
+} from '../../../../../../../common/cti/constants';
+import { DraggableBadge } from '../../../../../../common/components/draggables';
+import { FormattedFieldValue } from '../formatted_field';
+import { HorizontalSpacer } from './helpers';
+
+interface IndicatorDetailsProps {
+  contextId: string;
+  eventId: string;
+  indicatorDataset: string | undefined;
+  indicatorProvider: string | undefined;
+  indicatorReference: string | undefined;
+  indicatorType: string | undefined;
+}
+
+export const IndicatorDetails: React.FC<IndicatorDetailsProps> = ({
+  contextId,
+  eventId,
+  indicatorDataset,
+  indicatorProvider,
+  indicatorReference,
+  indicatorType,
+}) => (
+  <EuiFlexGroup
+    alignItems="flexStart"
+    data-test-subj="threat-match-indicator-details"
+    direction="row"
+    justifyContent="center"
+    gutterSize="none"
+    wrap
+  >
+    {indicatorType && (
+      <EuiFlexItem grow={false}>
+        <DraggableBadge
+          contextId={contextId}
+          data-test-subj="threat-match-indicator-details-indicator-type"
+          eventId={eventId}
+          field={INDICATOR_MATCHED_TYPE}
+          value={indicatorType}
+        />
+      </EuiFlexItem>
+    )}
+    {indicatorDataset && (
+      <>
+        <EuiFlexItem grow={false}>
+          <HorizontalSpacer>
+            <FormattedMessage
+              defaultMessage="from"
+              id="xpack.securitySolution.alerts.rowRenderers.cti.threatMatch.datasetPreposition"
+            />
+          </HorizontalSpacer>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <DraggableBadge
+            contextId={contextId}
+            data-test-subj="threat-match-indicator-details-indicator-dataset"
+            eventId={eventId}
+            field={INDICATOR_DATASET}
+            value={indicatorDataset}
+          />
+        </EuiFlexItem>
+      </>
+    )}
+    {indicatorProvider && (
+      <>
+        <EuiFlexItem grow={false} component="span">
+          <HorizontalSpacer>
+            <FormattedMessage
+              defaultMessage="provided by"
+              id="xpack.securitySolution.alerts.rowRenderers.cti.threatMatch.providerPreposition"
+            />
+          </HorizontalSpacer>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <DraggableBadge
+            contextId={contextId}
+            data-test-subj="threat-match-indicator-details-indicator-provider"
+            eventId={eventId}
+            field={INDICATOR_PROVIDER}
+            value={indicatorProvider}
+          />
+        </EuiFlexItem>
+      </>
+    )}
+    {indicatorReference && (
+      <>
+        <EuiFlexItem grow={false}>
```

Comment:
So then it would look like: 
```suggestion
        <IndicatorDetail>
```
?

There's not a huge benefit to that IMO, but we certainly could. I'm inclined to leave these dumb for now, at least until people start 👀 ing this row renderer and give UI feedback; premature DRYing often leads to unnecessary work.